backend/account/authentication_controller.py

```diff
@@ -259,6 +260,8 @@ def make_user_organization_display_name(self, user_name: str) -> str:
         return self.auth_service.make_user_organization_display_name(user_name)
 
     def user_logout(self, request: Request) -> Response:
+        session_id: str = request.COOKIES.get("sessionid")
```

Comment:
NIT: We should use constants for these if we aren't already. Also I suggest following either camel case (sessionId) or snake case convention here (session_id)